### PR TITLE
chore: update ibctest to latest commit

### PR DESCRIPTION
**Pull request by charleenfei**
This PR updates our e2e import of ibctest containing the [fix removing a redundant ValidateBasic check](https://github.com/strangelove-ventures/ibctest/pull/357) in BroadcastTx.

### Commit Message / Changelog Entry

```bash
chore: update ibctest to latest commit 
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/CONTRIBUTING.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/10-structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/CONTRIBUTING.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: e2e/tests/core/03-connection/connection_test.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 
 	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 	"github.com/strangelove-ventures/ibctest/v6/ibc"
-	"github.com/strangelove-ventures/ibctest/v6/test"
+	test "github.com/strangelove-ventures/ibctest/v6/testutil"
```
Comment (colin-axner) [inline e2e/tests/core/03-connection/connection_test.go:13]: In the future it might be nice to use a more descriptive alias. "test" is very generic

**Review by colin-axner — APPROVED** [review]
Yay! Thanks @charleenfei!

File: e2e/testconfig/testconfig.go [hunk 2]
```diff
@@ -166,6 +166,7 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string) ibc.Cha
 		},
 		Bin:            cc.Binary,
 		Bech32Prefix:   "cosmos",
+		CoinType:       "118",
```
Comment (damiannolan) [inline e2e/testconfig/testconfig.go:169]: I wonder if there's a constant for this in the sdk or gaia.. we can leave it til later

**Review by damiannolan — APPROVED** [review]
Thanks @charleenfei

File: e2e/testconfig/testconfig.go (hunk 2 above)
Comment (charleenfei) [reply e2e/testconfig/testconfig.go:169]: ah there is i think! i will use that one.

**Pull request merged**